<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VieSchedpp_Analyser</class>
 <widget class="QMainWindow" name="VieSchedpp_Analyser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0">
    <item>
     <widget class="QLabel" name="label_fileName">
      <property name="text">
       <string>fileName</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="skyCoverage">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QSplitter" name="splitter_skyCoverage">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QWidget" name="gridLayoutWidget">
           <layout class="QGridLayout" name="gridLayout_skyCoverage"/>
          </widget>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_skyCoverageLegend">
                <property name="text">
                 <string>legend</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
                <property name="flat">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_skyCoverageLayout">
                <property name="text">
                 <string>layout</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QTreeView" name="treeView_skyCoverage_sources">
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::MultiSelection</enum>
              </property>
              <property name="sortingEnabled">
               <bool>true</bool>
              </property>
              <property name="headerHidden">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_skyCoverageSourceFilter">
              <property name="placeholderText">
               <string>filter</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="world">
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_worldmap"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="sources">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_skymap"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="uv"/>
      <widget class="QWidget" name="statistics_general">
       <layout class="QVBoxLayout" name="verticalLayout_10" stretch="2,1">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>scans</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QSpinBox" name="spinBox_scans">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
                <property name="buttonSymbols">
                 <enum>QAbstractSpinBox::NoButtons</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>of</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBox_scans_total">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
                <property name="buttonSymbols">
                 <enum>QAbstractSpinBox::NoButtons</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>observations</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QSpinBox" name="spinBox_observations">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
                <property name="buttonSymbols">
                 <enum>QAbstractSpinBox::NoButtons</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>of</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBox_observations_total">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
                <property name="buttonSymbols">
                 <enum>QAbstractSpinBox::NoButtons</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>stations</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QSpinBox" name="spinBox_stations">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
                <property name="buttonSymbols">
                 <enum>QAbstractSpinBox::NoButtons</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>of</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBox_stations_total">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
                <property name="buttonSymbols">
                 <enum>QAbstractSpinBox::NoButtons</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>sources</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QSpinBox" name="spinBox_sources">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
                <property name="buttonSymbols">
                 <enum>QAbstractSpinBox::NoButtons</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>of</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBox_sources_total">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
                <property name="buttonSymbols">
                 <enum>QAbstractSpinBox::NoButtons</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>stations per scans</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="tableWidget_sta_per_scan">
              <row>
               <property name="text">
                <string>2 stations</string>
               </property>
              </row>
              <column>
               <property name="text">
                <string>scans</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>percent</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>scan durations</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="tableWidget_scan_duration">
              <row>
               <property name="text">
                <string>0-30 [s]</string>
               </property>
              </row>
              <column>
               <property name="text">
                <string>percent</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget_general">
          <row>
           <property name="text">
            <string>scans per station</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>scans per source</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>scans per baseline</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>average</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>std</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>median</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>min</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>max</string>
           </property>
          </column>
          <item row="0" column="0">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="1" column="0">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="2" column="0">
           <property name="text">
            <string/>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="statistics_station">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QTableWidget" name="tableWidget_statisticsStation"/>
          <widget class="QWidget" name="horizontalLayoutWidget_2">
           <layout class="QHBoxLayout" name="horizontalLayout_statisticsStation"/>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="statistics_source">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QTableWidget" name="tableWidget_statisticsSource"/>
          <widget class="QWidget" name="horizontalLayoutWidget_3">
           <layout class="QHBoxLayout" name="horizontalLayout_statisticsSource"/>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="statistics_baseline">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QSplitter" name="splitter_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QTableWidget" name="tableWidget_statisticsBaseline"/>
          <widget class="QWidget" name="horizontalLayoutWidget_4">
           <layout class="QHBoxLayout" name="horizontalLayout_statisticsBaseline"/>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_start">
         <item>
          <widget class="QLabel" name="label_start">
           <property name="text">
            <string>start</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSlider_start">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximum">
            <number>86400</number>
           </property>
           <property name="singleStep">
            <number>1800</number>
           </property>
           <property name="pageStep">
            <number>1800</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
           <property name="tickInterval">
            <number>3600</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateTimeEdit" name="dateTimeEdit_start">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="dateTime">
            <datetime>
             <hour>0</hour>
             <minute>0</minute>
             <second>0</second>
             <year>2000</year>
             <month>1</month>
             <day>1</day>
            </datetime>
           </property>
           <property name="displayFormat">
            <string>dd.MM.yy HH:mm:ss</string>
           </property>
           <property name="calendarPopup">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_end">
         <item>
          <widget class="QLabel" name="label_end">
           <property name="text">
            <string>stop</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSlider_end">
           <property name="maximum">
            <number>86400</number>
           </property>
           <property name="singleStep">
            <number>1800</number>
           </property>
           <property name="pageStep">
            <number>1800</number>
           </property>
           <property name="value">
            <number>86400</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBelow</enum>
           </property>
           <property name="tickInterval">
            <number>3600</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateTimeEdit" name="dateTimeEdit_end">
           <property name="displayFormat">
            <string>dd.MM.yy HH:mm:ss</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>duration</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_duration">
           <property name="suffix">
            <string> [s]</string>
           </property>
           <property name="maximum">
            <number>86400</number>
           </property>
           <property name="singleStep">
            <number>1800</number>
           </property>
           <property name="value">
            <number>86400</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>/</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_hours">
           <property name="suffix">
            <string> [h]</string>
           </property>
           <property name="maximum">
            <double>24.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.500000000000000</double>
           </property>
           <property name="value">
            <double>24.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_fixDuration">
           <property name="text">
            <string>fix duration</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menusky_coverage">
    <property name="title">
     <string>plots</string>
    </property>
    <addaction name="actionsky_coverage"/>
    <addaction name="actionworld_map"/>
    <addaction name="actionuv_coverage"/>
    <addaction name="actionsky_map"/>
   </widget>
   <widget class="QMenu" name="menustatistics">
    <property name="title">
     <string>statistics</string>
    </property>
    <addaction name="actiongeneral"/>
    <addaction name="actionper_station"/>
    <addaction name="actionper_source"/>
    <addaction name="actionper_baseline"/>
   </widget>
   <addaction name="menusky_coverage"/>
   <addaction name="menustatistics"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionsky_coverage">
   <property name="text">
    <string>sky coverage</string>
   </property>
  </action>
  <action name="actionuv_coverage">
   <property name="text">
    <string>uv coverage</string>
   </property>
  </action>
  <action name="actiongeneral">
   <property name="text">
    <string>general</string>
   </property>
  </action>
  <action name="actiontimes">
   <property name="text">
    <string>times</string>
   </property>
  </action>
  <action name="actionper_station">
   <property name="text">
    <string>per station</string>
   </property>
  </action>
  <action name="actionper_source">
   <property name="text">
    <string>per source</string>
   </property>
  </action>
  <action name="actionper_baseline">
   <property name="text">
    <string>per baseline</string>
   </property>
  </action>
  <action name="actionworld_map">
   <property name="text">
    <string>world map</string>
   </property>
  </action>
  <action name="actionsky_map">
   <property name="text">
    <string>sky map</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
